llvm/test/MC/RISCV/corev/simd/adddiv2-invalid.s

```diff
@@ -0,0 +1,21 @@
+# RUN: not llvm-mc -triple=riscv32 --mattr=+xcorevsimd %s 2>&1 \
```

Comment:
Consider merging the tests for those instructions that are identical in all but size. 
For example, `adddiv2.s` and `adddiv4.s` into `adddiv.s`. 
Another example would be that `cmpleub.s` and  `cmpleuh.s` could be merged into `cmpleu.s`. 
We will discuss this in the CORE-V LLVM Tools Engineering Call.

Reply:
Fixed

Reply:
All good.

Reply:
Fixed.

Reply:
@CharKeaney is happy this has been resolved.